fr-portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+<body>	
+	<header class="navbar">
+		<a href="#portfolio">Portfolio</a>
+		<a href="#about">About</a>
+		<a href="#contact">Contact</a>
+	</header>
+	<section class="welcome-banner" >
+							<h1>Freddie's Portfolio</h1>
+										
+	</section>	
+	<section id="about" class="about-alt">
+		<h2> About me </h2>
+			<div class="about-me">
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+				</div>
+				<div>
+					<p>Hi this is where I should put some information about myself</p>
+				</div>
+			</div>		
+	</section>
+		
+	<section id="portfolio" class="projects">
+		<h2>Portfolio</h2>
+			<div class="portfolio-container">
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+					<p>This is where my project information will go</p>
+				</div>
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+					<p>This is where my project information will go </p>
+				</div>	
+			</div>
+			<div class="portfolio-container">	
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+					<p>This is where my project information will go</p>
+				</div>	
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+					<p>This is where my project information will go</p>
+				</div>
+			</div>		
+	</section>
+	<section id="contact" class="contact-me">
+		<h2>Contact</h2>
+			<div class="my-contacts">
+				<div>
+					<h3>Email</h3>
+					<i class="fas fa-at"></i>
```

Comment:
I think you may have got mixed up with `<a>` and `<i>` tags, the former is an anchor that is used to provide a link whereas the latter is used to style text, like making it italic

Reply:
i class has been used to add the icons. It is linked to fontawesome CSS